src/core/sys/windows/stacktrace.d

```diff
@@ -345,26 +345,38 @@ extern(Windows) BOOL FixupDebugHeader(HANDLE hProcess, ULONG ActionCode,
     return FALSE;
 }
 
-private string generateSearchPath()
+private char[] generateSearchPathAlloc()
```

Comment:
if the point is to avoid GC allocation, please mark is `@nogc` so it is obvious.